tbtadm/controller.cpp

```diff
@@ -653,6 +655,188 @@ catch (...)
     m_err << "Unknown exception\n";
 }
 
+void tbtadm::Controller::addToBootACL(const fs::path& dir, std::string uuid)
+{
+    auto fname = dir.filename().string();
+    auto num = fname.substr(0, fname.find('-'));
+    auto boot_acl_path = sysfsDevicesPath / (domain + num) / boot_acl;
+
+    if (!fs::exists(boot_acl_path))
+    {
+        /* Cannot find boot_acl */
+        return;
+    }
+
+    /* Read boot_acl */
+    std::stringstream stream(readAndTrim(boot_acl_path));
+    /* Check maximum allowed boot_acl string size */
+    if (stream.str().size() > 36 * 16 + 1 * 15)
+    {
+        m_out << "boot_acl contains too much characters\n";
+        return;
+    }
+
+    std::queue<std::string> acl_queue;
+
+    while(stream.good())
```

Comment:
The more common pattern is:
```
std::string substring;
while(getline(stream, substring, ','))
...
```
As `getline` returns the stream and stream is convertible to `bool` and it's `true` only if it's `good`.